src/base/utils/misc.cpp

```diff
@@ -602,6 +602,26 @@ QString Utils::Misc::zlibVersionString()
 }
 
 #ifdef Q_OS_WIN
+bool Utils::Misc::applyMarkOfTheWeb(const Path &file)
+{
+    const QString zoneIDStream = file.toString() + u":Zone.Identifier";
+    HANDLE handle = ::CreateFileW(zoneIDStream.toStdWString().c_str(), GENERIC_WRITE
```

Comment:
What if the target file system does not support alternative file streams? I hope this action will be silently skipped (for example, when trying to open a file stream), so that the user will not receive annoying/confusing error messages, etc.

Reply:
I tried to download to a FAT32 drive and there are no error whatsoever. The torrent is downloaded successfully without any signs of failure.